### PR TITLE
Add secure paging support for plat-ti

### PR DESCRIPTION
**Pull request by glneo**

### CODE REVIEW

File: core/arch/arm/plat-ti/main.c [hunk 1]
```diff
@@ -50,6 +50,7 @@
 static struct gic_data gic_data;
 static struct serial8250_uart_data console_data __early_bss;
 
+register_phys_mem(MEM_AREA_RAM_SEC, TZDRAM_BASE, CFG_TEE_RAM_VA_SIZE);
```
Comment (jforissier) [inline core/arch/arm/plat-ti/main.c]: Is this needed? And, if so, is this needed for this purpose ("Add secure paging support"), or should it be part of another patch? I'm wondering because `TZDRAM_BASE` is used already.

Comment (glneo) [inline core/arch/arm/plat-ti/main.c]: This is needed, when building with CFG_WITH_PAGER the "tee-pageable.bin" is loaded to this area.

**Comment by glneo** [comment]
Updated with Acked-by

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`
